pubsub/google/cloud/pubsub_v1/subscriber/policy/base.py

```diff
@@ -65,6 +66,10 @@ class BasePolicy(object):
     """
 
     _managed_ack_ids = None
+    _IDEMPOTENT_RETRIES = (
```

Comment:
I hate to be *that guy*, but I'm not a fan of this name, how about `_RETRYABLE_STREAM_ERRORS`?